### PR TITLE
TEVA-2162 - Make alerts less noisy; Split app and worker CPU

### PR DESCRIPTION
**Pull request by nickromneydfe**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-2162

## Changes in this PR:

- Increase threshold on certain alerts
- Split out worker from app for CPU usage

### CODE REVIEW

File: terraform/monitoring/config/alert.rules.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ groups:
   rules:
   - alert: ProdRequestsFailuresElevated
     # Condition for alerting
-    expr: sum(rate(requests{app="teaching-vacancies-production", status_range=~"0xx|4xx|5xx"}[5m]))*60 > 25
+    expr: sum(rate(requests{app="teaching-vacancies-production", status_range=~"0xx|4xx|5xx"}[5m]))*60 > 40
```
Comment (saliceti) [inline terraform/monitoring/config/alert.rules.yml:6]: 25 looks reasonable?

Comment (nickromneydfe) [reply terraform/monitoring/config/alert.rules.yml]: Yes, I agree. Have left it at 25. 
Looks like elsewhere we've identified the source of 4 additional HTTP failures with missing files

File: terraform/monitoring/config/alert.rules.yml [hunk 2]
```diff
@@ -28,6 +28,17 @@ groups:
     labels:
       environment: production
       severity: high
+  - alert: ProdWorkerCPUHigh
+    expr: avg by ( app ) (cpu{app=~"teaching-vacancies-worker-production"}) > 75
+    for: 5m
+    annotations:
+      summary: Worker CPU usage high
+      description: "CPU usage at least 60% for more than 5 minutes (current value: {{ $value }})"
```
Comment (csutter) [inline terraform/monitoring/config/alert.rules.yml:36]: Copypasta 🍝

**Comment by csutter** [comment]
Looks great – although maybe we can increase the time threshold for the worker app too? I could imagine 75%+ CPU on the worker for 5 minutes might still happen quite frequently...

File: terraform/monitoring/config/alert.rules.yml (hunk 2 above)
Comment (nickromneydfe) [reply terraform/monitoring/config/alert.rules.yml]: - Corrected description for split out alerts
- Updated documentation

**Review by saliceti — APPROVED** [review]

**Pull request merged**